### PR TITLE
Feature/loading and error

### LINKED ISSUE #23: FEATURE: Error Page
**Is your feature request related to a problem? Please describe.**

- we need a page to reroute to when there is an error. 
- It should display a message telling the user that there was an error 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Additional context**

### LINKED ISSUE #30: FEATURE: Loading component
**Describe the solution you'd like**

- a `Loading` component to render while pages are loading 

**Additional context**

- we can use the `Font Awesome` icons that we have already imported to add some flair to the loading page

### PR DESCRIPTION
**Pull request by richardltyler**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- adds a `NotFound` page for error-handling
- adds a `Loading` component 

## *Instructions for reviewer:*
*Where should the reviewer start?*
*How should this be manually tested?*
1) try to visit a path that is not declared to view the `NotFound` component
2) comment out the import and `Route` declaration for `Loading` in `App.js` and visit `/loading` to see the `Loading` component

*Any background context you want to provide?*
*What are the relevant tickets?*
- closes #30 
- closes #23 

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
- `Loading` component still needs to be implemented properly 
- currently, `Loading` is set up with a path that is probably not the best way to handle the component 

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/common/Loading.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCompactDisc } from '@fortawesome/free-solid-svg-icons'
+import './common.css';
+
+const Loading = () => {
+  return (
+    <div className='Loading-container'>
+      <h2 className='Loading-heading'>Loading...</h2>
+      <FontAwesomeIcon icon={faCompactDisc} className='fa-spin' size='5x' />
+    </div>
```
Comment (kristenmb) [inline src/components/common/Loading.js:11]: This looks great! Thanks for sorting out a spinner

**Review by kristenmb — APPROVED** [review]
This looks great thanks for tackling these!

**Review by kevinhartmann23 — APPROVED** [review]
Nice work, thanks for handling errors and loading!

File: src/components/common/Loading.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCompactDisc } from '@fortawesome/free-solid-svg-icons'
```
Comment (DrewBradley) [inline src/components/common/Loading.js:3]: Neat!

File: src/components/common/Loading.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import './common.css';
+
+const Loading = () => {
+  return (
+    <div className='Loading-container'>
+      <h2 className='Loading-heading'>Loading...</h2>
+      <FontAwesomeIcon icon={faCompactDisc} className='fa-spin' size='5x' />
```
Comment (DrewBradley) [inline src/components/common/Loading.js:10]: NOICE!!! Heck yes with that animation!

**Review by DrewBradley — APPROVED** [review]
Gorgeous!

**Pull request merged**